src/Microsoft.ML.Data/Transforms/NormalizeColumnDbl.cs

```diff
@@ -597,6 +597,7 @@ internal static void GetMedianSoFar(in double num, ref double median, ref MaxHea
     /// It tracks median values of non-sparse values (vCount).
     /// NaNs are ignored when updating min and max.
     /// </summary>
+    [BestFriend]
```

Comment:
Doesn't seem to be used. Revert?

Reply:
It is used in SeasonalityDetector (otherwise won't compile)